week-5/A-dom-manipulation/exercise.js

```diff
@@ -13,7 +30,22 @@ Task 2
 
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
+///////////METHOD 1////////////
+// var alertBtn = document.querySelector("#alertBtn");
+// alertBtn.addEventListener("click", alertMessage);
 
+// function alertMessage() {
+//     alert("Red light");
+//     alertBtn.style.backgroundColor = "red";
+// }
+
+/////////////METHOD 2///////////////
+divButtonChild[1].addEventListener("click", alertMessage);
+
```

Comment:
Imagine your boss asks you to reorder the buttons, or add a new one at the start.  Suddenly you have to change many lines of code!